### DISCUSSION TITLE
Configuração de campos da po-page-dynamic-table ocupa muito espaço

### DISCUSSION THREAD

**Discussion by gtrabach** [po-ui/po-angular#1768] [Tire suas dúvidas] [answered]
Atualizei o meu PO UI para a versão 15 e percebi que o comportamento da configuração de campos do componente po-page-dynamic-table mudou.

Anteriormente, era uma engrenagem junto as colunas da tabela, agora é uma engrenagem acima da tabela e abaixo da lupa de pesquisa.

Este comportamento é esperado? Existe alguma forma de voltar ao comportamento anterior?

Da forma que está agora, a lupa acaba ocupando um espaço considerável da tela, deixando o layout mais poluído e ocupando o espaço que poderia ser de uma linha da tabela.

Antes:
![image](https://github.com/po-ui/po-angular/assets/46979335/16e01b94-4117-4660-80f4-1dfc48070357)

Depois:
![image](https://github.com/po-ui/po-angular/assets/46979335/0b936b41-e30d-4573-b754-c56dfc0c3a0a)

Grato.

**Comment by anliben** [comment, marked as answer]
Boa Tarde @gtrabach, tudo bem?

O componente passou por uma implementação do animaliaDS e segue as diretrizes do nosso Design System que realizaram estudos e conseguiram chegar em um conceito que fosse, minimalista, acessível, consistente. Este comportamento que esta querendo não pode ser customizado!

Attr.
